apps/pi-drone-detector/scripts/download_datasets.py

```
@@ -0,0 +1,316 @@
+    print()
+    return dest
+
+
+def extract_archive(archive: Path, dest: Path):
+    """Extract zip or tar archive."""
+    print(f"  Extracting {archive.name}...")
+    dest.mkdir(parents=True, exist_ok=True)
+
+    if archive.suffix == '.zip':
+        with zipfile.ZipFile(archive, 'r') as zf:
+            zf.extractall(dest)
```

Ruff (0.14.10)

Uses of `tarfile.extractall()`

(S202)